style.css

```
@@ -9,6 +9,38 @@ body {
     overflow-x: hidden;
 }
 
+/* scroll-up button starts */
+
+#btnScrollToTop {
+    position: fixed;
+    right: 10px;
+    bottom: 10px;
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    background: #8614f8;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
+    color: #FFFFFF;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    z-index: 1;
+}
+
+#btnScrollToTop:hover {
+    width: 52px;
+    height: 52px;
```

```suggestion
    width: 50px;
    height: 50px;
```
I suggest not to increase the height and width here or keep it same above too.
else it doesn't looks good

---

Thankyou😊.( @MrKrishnaAgarwal)